cwms-data-api/src/main/java/cwms/cda/api/LocationGroupController.java

```diff
@@ -100,7 +104,8 @@ public void getAll(@NotNull Context ctx) {
             DSLContext dsl = getDslContext(ctx);
             LocationGroupDao cdm = new LocationGroupDao(dsl);
 
-            String office = ctx.queryParam(OFFICE);
+            String groupOfficeId = requiredParam(ctx, OFFICE);
```

Comment:
While the swagger API should declare this as required, this would break backwards compatibility and require client updates. Leave swagger change and then put a comment in the code explaining why it isn't marked required.

Reply:
Swapped to not required and added a comment explaining why.